backward.hpp

```diff
@@ -1670,7 +1644,18 @@ class TraceResolverLinuxImpl<trace_resolver_tag::libdwarf>
       return trace;
     }
 
-    trace.object_filename = symbol_info.dli_fname;
+    if(symbol_info.dli_fname == get_argv0()) {
```

Comment:
this code looks like a duplicate from above. do you think we can avoid duplication here?